cookbooks/aws-parallelcluster-scheduler-plugin/templates/default/compute_fleet_status/update-compute-fleet-status.erb

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+_error_exit() {
+  echo "ERROR: $1"
+  exit 1
+}
+
+_help() {
+    local -- _cmd=$(basename "$0")
+
+  cat << EOF
+
+  Usage: ${_cmd} [OPTION]...
+
+  Copy the AWS ParallelCluster Package to an S3 bucket.
+
+  --status                      Expected status of compute fleet
+  -h, --help                    Print this help message
+EOF
+}
+
+main() {
+    # parse input options
+    while [ $# -gt 0 ] ; do
+        case "$1" in
+            --status)           _status="$2"; shift;;
+            --status=*)         _status="${1#*=}";;
+            -h|--help|help)     _help; exit 0;;
+            *)                  _help; echo "[error] Unrecognized option '$1'"; exit 1;;
+        esac
+        shift
+    done
+
+    # verify required parameters
+    if [ -z "${_status}" ]; then
+        _error_exit "--status parameter not specified"
+        _help;
+    fi
+
```

Comment:
please add a check on _status value, so that the error message is not coming from compute_fleet_status.py